pkgs/tools/package-management/nvd/default.nix

```diff
@@ -0,0 +1,30 @@
+{ fetchFromGitLab, lib, python3, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "nvd";
+  version = "0.1.1";
+
+  src = fetchFromGitLab {
+    owner = "khumba";
+    repo = "${pname}";
+    rev = "refs/tags/v${version}";
```

Comment:
I think `"v${version}"` is good enough, no need for `"refs/tags/"`

Reply:
```suggestion
    rev = "v${version}";
```